OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -507,7 +507,7 @@ bool LoadAsset(IReadOnlyPackage package, string filename)
 					currentSprites = world.Map.Rules.Sequences.SpriteCache[prefix + filename];
 					currentFrame = 0;
 
-					if (frameSlider != null)
+					if (frameSlider != null && currentSprites?.Length > 0)
```

Comment:
wouldn't this crash if currentSprites was null? Or does it accept `null > 0`. I'm wondering how this compiles